Java/core/src/main/java/com/amazon/randomcutforest/returntypes/SampleSummary.java

```diff
@@ -83,4 +92,62 @@ public SampleSummary(double[] point) {
         this.deviation = new float[point.length];
     }
 
+    public SampleSummary(int dimension, float[][] typicalPoints, float[] relativeLikelihood) {
+        this.addTypical(dimension, typicalPoints, relativeLikelihood);
+    }
+
+    public void addTypical(int dimension, float[][] typicalPoints, float[] relativeLikelihood) {
```

Comment:
If we are assigning `typicalPoints` to the `summaryPoints` array, why don't we just call the method argument `summaryPoints`?

Reply:
Done